aot/src/main/java/com/dylibso/chicory/aot/AotOpCode.java

```diff
@@ -0,0 +1,237 @@
+package com.dylibso.chicory.aot;
+
+import com.dylibso.chicory.wasm.types.OpCode;
+import java.util.EnumMap;
+import java.util.Map;
+import java.util.Optional;
+
+enum AotOpCode {
+    LABEL,
+    DROP_KEEP,
```

Comment:
What's `DROP_KEEP`? What's the semantics?

Reply:
The WABT IR has the same instruction. I've been thinking that `UNWIND` might be a better name as that describes the purpose and usage rather. The semantics can be seen in `AotEmitters.DROP_KEEP()`. What it does is to drop `x` number of intermediate stack values, keeping the last `y` values on the stack. This is how unwinds work.